PROTOCOL.md

```diff
@@ -260,6 +260,12 @@ etc). `NAME` is the nickname of the Pokemon performing the action.
 > `switch` means it was intentional, while `drag` means it was unintentional
 > (forced by Whirlwind, Roar, etc).
 
+`|swap|POKEMON|POSITION|CAUSE`
+
+> Moves already active `POKEMON` to active field `POSITION` where `0` is the
+> leftmost position and each position to the right increments by 1. `CAUSE`
+> is optional and only used for debug.
```

Comment:
It shows up for Ally Switch but doesn't do anything aside from showing up.

(though as a side note it could probably be used to fix the graphical glitch caused by Ally Switch)